gi-docgen.yaml

```diff
@@ -1,10 +1,22 @@
 package:
   name: gi-docgen
   version: "2024.1"
-  epoch: 1
+  epoch: 2
   description: A documentation generator for GObject-based libraries
   copyright:
     - license: Apache-2.0 OR GPL-3.0-or-later
+  dependencies:
+    provider-priority: 0
+
+vars:
+  pypi-package: gi-docgen
+
+data:
+  - name: py-versions
+    items:
+      3.11: '311'
+      3.12: '312'
+      3.13: '313'
```

Comment:
We are not building for python3.10 because of [toml version constrain ](https://gitlab.gnome.org/GNOME/gi-docgen/-/blob/main/meson.build#L42)

We are getting the error
```
meson.build:45:6: ERROR: Problem encountered: tomli 1.0 or newer required on Python 3.10 and older subpackage=py3.10-gi-docgen
```